### ISSUE TITLE
Carousel: autoplay should not trigger live region

### ISSUE THREAD

**Issue by ianmcburnie** [eBay/ebayui-core#296]
# Bug Report

## Description

The live region should not announce when autoplay is enabled. This is distracting and creates a barrier for users of assistive technology.

To simplify things, I propose permanently dropping role="status" from the markup (for all variations of carousel) then we can simply flip aria-live between a value of `off` and `polite` based on whether playback is active or not.

The reason we had role of status was to double up on support for those screen readers that historically do not support aria-live, but support is good enough now that I think we can drop it.

We also need an additional span tag inside live region to support voiceover (see #294). We may as well fix this here too if it's low risk (leave it out if not).

Current markup:

```html
<span id="carousel-status-c1-1" class="clipped" role="status" aria-live="polite">
Showing Slide 3 of 3 - Carousel
</span>
```

Propose markup when autoplaying/playing:

```html
<span id="carousel-status-c1-1" class="clipped" aria-live="off">
  <span>Showing Slide 3 of 3 - Carousel</span>
</span>
```

Propose markup when not playing/autoplaying:

```html
<span id="carousel-status-c1-1" class="clipped" aria-live="polite">
  <span>Showing Slide 3 of 3 - Carousel</span>
</span>
```

## Workaround
No.

## Context

`Normally, only aria-live="polite" is used. Any region which receives updates that are important for the user to receive, but not so rapid as to be annoying, should receive this attribute. The screen reader will speak changes whenever the user is idle.`

`For regions which are not important, or would be annoying because of rapid updates or other reasons, silence them with aria-live="off".`

**Comment by ianmcburnie** [comment]
Fixed and merged.
